tests/data_tests/test_samplers.py

```diff
@@ -4,7 +4,8 @@
 
```

Comment:
Can you use https://docs.python.org/3/library/unittest.html in coherence with the rest of the tests?

Reply:
Done